src/Proto3/Suite/Class.hs

```diff
@@ -583,7 +583,9 @@ instance (MessageField e, KnownSymbol comments) => MessageField (e // comments)
 decodePacked
   :: Parser RawPrimitive [a]
   -> Parser RawField (PackedVec a)
-decodePacked p = Parser $ \fs -> fmap (fromList . join . F.toList) $ TR.sequence $ fmap (runParser p) fs
+decodePacked = Parser
```

Comment:
I think some of your other point-free and `fmap` -> `<$>` changes in this file are reasonable and maintain readability (if not increase it in certain cases); however, I don't think that is the case with the change on this line. This required more attention from me to penetrate its meaning.

Reply:
This was also difficult for me to read.  Actually, even the original code before your change was difficult for me to read, too :)

I did like the simplification of `sequence . fmap f → traverse f`, but overall I still think this is too point-free

Also, independent of point-free versus point-ful, I think another way to improve the clarify of this code is to split the `fromList . join . F.toList` out into a named function, like this:

```haskell
decodePacked = ...
  where
    adapt :: Seq [a] -> Vector a
    adapt = fromList . join . F.toList
```

Reply:
Yeah, I just did this via rewrite rules, I'm not sure what's going on either.

Reply:
Factored out a local function.